update_python_spec.lisp

```diff
@@ -1,21 +1,12 @@
 #!/usr/bin/env sbcl --script
 (load "~/.sbclrc")
 (ql:quickload :rpcq)
+(rpcq::clear-messages)
 
-; messages
-(rpcq::clear-messages :messages)
-(load "src/messages.lisp")
-(with-open-file (f "rpcq/messages.py"
-                   :direction ':output
-                   :if-exists ':supersede)
-  (rpcq::python-message-spec f (gethash :messages rpcq::*messages*))
-  (write-line "Wrote new messages.py"))
-
-; core-messages
-(rpcq::clear-messages :core-messages)
-(load "src/core-messages.lisp")
-(with-open-file (f "rpcq/core-messages.py"
-                   :direction ':output
-                   :if-exists ':supersede)
-  (rpcq::python-message-spec f (gethash :core-messages rpcq::*messages*))
-  (write-line "Wrote new core-messages.py"))
+(dolist (namespace (list "messages" "core-messages"))
+  (load (format nil "src/~A.lisp" namespace))
```

Comment:
Welcome to the world where pathnames are objects. One can do:
```
(make-pathname :name namespace :type "lisp" :directory '(:RELATIVE "src"))
```

One can also do the slightly related:
```
(merge-pathnames (make-pathname :name namespace :type "lisp") "src/")
```
but it's not needed here.